File: DayOne/Java/LevelOne/PalindromicArrayTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

public class PalindromicArrayTest{

    @Test
    public void testThatIhaveAnArrayAndReturnFalse(){

    int [] array = {45, 0, 8, 0, 1};

    boolean arrayCheck = PalindromicArray.palindromeCheck(array);

    assertFalse(arrayCheck);
      
    }

    @Test
    public void testThatIhaveAnArrayAndReturnTrue(){

    int [] array = {45, 0, 8, 0, 45};

    boolean arrayCheck = PalindromicArray.palindromeCheck(array);

    assertTrue(arrayCheck);
      
    }
    

}

File: DayOne/Python/LevelOne/palindromic_array_test.py
from unittest import TestCase

from palindromic_array import *

class TestPalindromicArray(TestCase):

    def test_that_an_array_is_a_palindrome_returns_false(self):

        array = [45, 0, 8, 0, 1]

        array_check = palindrome_check(array)

        self.assertFalse(array_check)

    def test_that_an_array_is_a_palindrome_returns_true(self):

        array = [45, 0, 8, 0, 45]

        array_check = palindrome_check(array)

        self.assertTrue(array_check)
